buildlib/tools/builds.sh

```diff
@@ -211,13 +211,19 @@ build_cuda() {
 		if az_module_load $GDRCOPY_MODULE
 		then
 			echo "==== Build with enable cuda, gdr_copy ===="
-			${WORKSPACE}/contrib/configure-devel --prefix=$ucx_inst --with-cuda --with-gdrcopy
+			${WORKSPACE}/contrib/configure-release --prefix=$ucx_inst --with-cuda --with-gdrcopy
 			$MAKEP
 			make_clean distclean
 
-			${WORKSPACE}/contrib/configure-release --prefix=$ucx_inst --with-cuda --with-gdrcopy
+			# Use path to CUDA instead of loading CUDA as module, to check
+			# GDRCopy subcomponent does not include CUDA headers
+			cuda_path=$(dirname $(dirname $(which nvcc)))
+			az_module_unload $CUDA_MODULE
+			${WORKSPACE}/contrib/configure-devel --prefix=$ucx_inst --with-cuda=$cuda_path --with-gdrcopy
 			$MAKEP
 			make_clean distclean
+			az_module_load $CUDA_MODULE
```

Comment:
IMO would be easier to move the new command lines after line 235 below, when cuda is already unloaded

Reply:
ok